lib/dotcom_web/components/live_components/trip_planner_form.ex

```diff
@@ -129,11 +129,6 @@ defmodule DotcomWeb.Components.LiveComponents.TripPlannerForm do
                   </.inputs_for>
                 </div>
               </:content>
-              <:extra :if={used_input?(f[:modes])}>
```

Comment:
Can we undo this removal? This removes the "please select a mode" message if all modes are left unchecked, and this change doesn't seem to have anything to do with the rest of this PR anyway.

Reply:
Ah, gotcha.

![image](https://github.com/user-attachments/assets/3c144696-77ef-485c-a31a-3129f8ba8f7f)

As I just described: "Please select at least one mode of transit". But this could just as easily be included as a `<div>` after the accordion instead of using the prior `extra` attribute.